roms_tools/setup/nesting.py

```diff
@@ -77,11 +82,17 @@ class ChildGrid(Grid):
 
     def __post_init__(self):
         super().__post_init__()
-        self._map_child_boundaries_onto_parent_grid_indices()
-        self._modify_child_topography_and_mask()
+        self._map_child_boundaries_onto_parent_grid_indices(verbose=self.verbose)
+        self._modify_child_topography_and_mask(verbose=self.verbose)
```

Comment:
it doesn't make a lot of sense to have `self.verbose` be a member variable and then pass it as a parameter - isn't it already available?

Reply:
But there is another method `update_topography` that also calls `_map_child_boundaries_onto_parent_grid_indices` and `update_topography` has the `verbose` arguments. 

The user could do the following:
1. `ChildGrid(verbose=True, ...)`
2. `ChildGrid.update_topography(verbose=False)`

Reply:
understood